data/deep missions.txt

```diff
@@ -328,8 +328,9 @@ mission "Deep: TMBR 0"
 				`	"No thanks. I have better things to do."`
 					decline
 			
-			`	All five scientists burst out in a cheer, clearly excited that they will be able to see their favorite band live. After the cheering dies down, Laura says "We must get going now, Captain. We'll be on Midgard for the next few months, so if you find 'There Might Be Riots,' please bring them there." Each member of the group says goodbye to you as they board their Bounder and leave for Midgard.`
-			`	If you are going to look for the band "There Might Be Riots," you should go to <destination> where you last saw them.`
+			`	All five scientists burst out in a cheer, clearly excited for the chance to see their favorite band live. As the cheering dies down Laura says, "We must get going now, <first>. We're being stationed on Midgard for the next few months, so if you do find 'There Might Be Riots,' please bring them there." Each member of the group says goodbye to you as they depart.`
+			``
+			`	(If you are going to look for the band "There Might Be Riots," you should go to <destination>, where you last saw them.)`
```

Comment:
Don't like the fourth wall breaking in such a minor mission (pretty sure the only 4th wall breaks in game are during the tutorial, right before you enter Pug space in the FW storyline, and during one of the epilogues if you choose a certain choice).